.github/workflows/build-magma-linux.yml

```diff
@@ -30,10 +30,10 @@ jobs:
     runs-on: linux.2xlarge
     strategy:
       matrix:
-        cuda_version: ["124", "121", "118"]
+        cuda_version: ["118", "121", "124"]
```

Comment:
What is the benefit of changing the order?

Reply:
None! it was just trying to be consistent with increasing versions left-to-right